### PR TITLE
Update tutorial-org-aa.yml

### PR DESCRIPTION
**Pull request by CLYVR**
Clarifying how and when the # / Back feature works.

Note that the following diagrams also need to be updated.

Diagram of Auto Attendants at a lawyer's office
o	Main Menu - remove the # key reference
o	Remove the line/arrow and circle with the # key reference from the following:
	Billing 
	Hours&Location
	MoreOptions
	Lawyers
	Paralegals
	StillMoreOptions

Another diagram of Auto Attendants at a lawyer's office

o	Main Menu - remove the # key reference
o	Remove the line/arrow and circle with the # key reference from the following:
	Billing 
	Hours&Location
	MoreOptions
	Paralegals
	StillMoreOptions

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/tutorial-org-aa.yml [hunk 1]
```diff
@@ -249,36 +253,36 @@ items:
     For hours and location, press 3.
     For more options, press 4.
 
-    *Available options not mentioned: zero, \*, and \#.*
+    *Available options not mentioned: zero, and \*.*
```
Comment (JohanFreelancer9) [inline Teams/tutorial-org-aa.yml:256]:
```suggestion
    *Available options not mentioned: zero and \*.*
```

File: Teams/tutorial-org-aa.yml [hunk 2]
```diff
@@ -249,36 +253,36 @@ items:
...
 
     #### Script 4: billing
     To speak to Jakob Roth, press 1.
     To speak to Laura Heinrich, press 2.
 
-    *Available options not mentioned: zero, \*, and \#.*
+    *Available options not mentioned: zero, and \*.*
```
Comment (JohanFreelancer9) [inline Teams/tutorial-org-aa.yml:262]:
```suggestion
    *Available options not mentioned: zero and \*.*
```

File: Teams/tutorial-org-aa.yml [hunk 3]
```diff
@@ -249,36 +253,36 @@ items:
...
 
      This script could be omitted if you redesigned using Call Queues.
 
     #### Script 5: hours and Location
     We are open for business from 8:30am to 5:30 pm, Tuesday through Saturday at 1234 Green Street near Central avenue, Planktown Pennsylvania,
```
Comment (JohanFreelancer9) [inline Teams/tutorial-org-aa.yml:267]:
```suggestion
    We are open for business from 8:30 am to 5:30 pm, Tuesday through Saturday at 1234 Green Street near Central avenue, Planktown Pennsylvania, 
```

File: Teams/tutorial-org-aa.yml [hunk 4]
```diff
@@ -249,36 +253,36 @@ items:
...
+    *Available options not mentioned: zero, and \*.*
 
      This script could be omitted if you redesigned using Call Queues.
 
     #### Script 5: hours and Location
     We are open for business from 8:30am to 5:30 pm, Tuesday through Saturday at 1234 Green Street near Central avenue, Planktown Pennsylvania, 
     directly across from City Hall.
 
-    Available options not mentioned: zero, *, and #.
+    *Available options not mentioned: zero, and \*.*
```
Comment (JohanFreelancer9) [inline Teams/tutorial-org-aa.yml:270]:
```suggestion
    *Available options not mentioned: zero and \*.*
```

File: Teams/tutorial-org-aa.yml [hunk 5]
```diff
@@ -249,36 +253,36 @@ items:
...
 
     #### Script 6: more menu options
     To speak to one of the lawyers, press 1.
     To speak to the office manager, press 2. 
     To speak to one of the paralegals, press 3.
     For other options, press 4.
 
-    *Available options not mentioned: zero, \*, and \#.*
+    *Available options not mentioned: zero, and \*.*
```
Comment (JohanFreelancer9) [inline Teams/tutorial-org-aa.yml:278]:
```suggestion
    *Available options not mentioned: zero and \*.*
```

File: Teams/tutorial-org-aa.yml [hunk 6]
```diff
@@ -249,36 +253,36 @@ items:
...
 
     #### Script 7: lawyer menu
     To speak to Olga Gerasimova, press 1.
     To speak to for Manuela Carstens, press 2.
```
Comment (JohanFreelancer9) [inline Teams/tutorial-org-aa.yml:282]:
```suggestion
    To speak to Manuela Carstens, press 2.
```

File: Teams/tutorial-org-aa.yml [hunk 7]
```diff
@@ -249,36 +253,36 @@ items:
...
     To speak to for Andrey Alexeeva, press 3.
```
Comment (JohanFreelancer9) [inline Teams/tutorial-org-aa.yml:283]:
```suggestion
    To speak to Andrey Alexeeva, press 3.
```

File: Teams/tutorial-org-aa.yml [hunk 8]
```diff
@@ -249,36 +253,36 @@ items:
...
 
-    *Available options not mentioned: zero, \*, and \#.* 
+    *Available options not mentioned: zero, and \*.*
```
Comment (JohanFreelancer9) [inline Teams/tutorial-org-aa.yml:285]:
```suggestion
    *Available options not mentioned: zero and \*.*
```

File: Teams/tutorial-org-aa.yml [hunk 9]
```diff
@@ -290,15 +294,15 @@ items:
     To speak to Dariya Polyakova, press 5.
     To speak to Milana Nikolaeva, press 6.
 
-    *Available options not mentioned: zero, \*, and \#.*
+    *Available options not mentioned: zero, and \*.*
```
Comment (JohanFreelancer9) [inline Teams/tutorial-org-aa.yml:297]:
```suggestion
    *Available options not mentioned: zero and \*.*
```

File: Teams/tutorial-org-aa.yml [hunk 10]
```diff
@@ -290,15 +294,15 @@ items:
...
 
      This script could be omitted if you redesigned using Call Queues.
 
     ####	Script 9: Other menu options
     To speak to our investigator, press 1.
     To request a copy of your records, press 2.
 
-    *Available options not mentioned: zero, \*, and \#.*
+    *Available options not mentioned: zero, and \*.*
```
Comment (JohanFreelancer9) [inline Teams/tutorial-org-aa.yml:305]:
```suggestion
    *Available options not mentioned: zero and \*.*
```

**Comment by scanum** [comment]
@dstrome  PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**